src/index.css

```
@@ -49,11 +49,17 @@ limitations under the License.
   /* The distance to inset non-full-width content from the edge of the window
   along the inline axis */
   --inline-content-inset: max(var(--cpd-space-4x), calc((100vw - 1180px) / 2));
+
+  --background-gradient: url("graphics/backgroundGradient.svg");
 }
 
 .cpd-theme-dark {
   --stopgap-color-on-solid-accent: var(--cpd-color-text-primary);
   --stopgap-background-85: rgba(16, 19, 23, 0.85);
+
+  background-size: calc(max(100%, 100vw)) calc(max(100%, 100vh));
+  background-repeat: no-repeat;
+  background-position: bottom right;
```

After playing around with these parameters a bit, I think the following gets a little bit closer to how the graphic is sized and placed in Figma:

```suggestion
  background-size: calc(max(1440px, 100vw)) calc(max(800px, 100vh));
  background-repeat: no-repeat;
  background-position: center;
```